config/peerpods/podvm/libvirt-podvm-image-handler.sh

```diff
@@ -73,6 +72,19 @@ function create_libvirt_image() {
 # Function to dowload the rhel base image
 
 function download_rhel_kvm_guest_qcow2() {
+    #Validate RHEL version for IBM Z Secure Enablement
+    if [ "$SE_BOOT" == "true" ]; then
+        version=$(echo $BASE_OS_VERSION | awk -F "." '{ print $1 }')
+        release=$(echo $BASE_OS_VERSION | awk -F "." '{ print $2 }')
+        if [ "$version" -lt 9 ]; then
+            error_exit "Libvirt Secure Execution supports RHEL OS version 9.4 or above"
+        else
```

Comment:
nit: you can do it in the same conditional with **or**

Reply:
@snir911  Yes, we can achieve this with "or" but only concern is when "version" is 10 and release is less than 4 . This will break in future .

Reply:
@ANJANA-A-R-K  when version is 10 and release is less than 4. This condition will break.

Reply:
Modified the condition